examples/prompt.cpp

```diff
@@ -1,11 +1,23 @@
 #include <cpp-terminal/terminal.h>
 
 #include <vector>
+#include <functional>
+#include <string>
 
 using Term::Terminal;
 using Term::prompt;
 
 
+bool determine_completeness(std::string command)
+{
+    // Determine if the statement is complete
+    // Get the tokens
+    bool complete;
+    complete = true;
+    return complete;
```

Comment:
You can look if the string ends with `\` (as in Python) and if it does, it will not be complete. That way you get rid of the warning and show how to use it at the same time.

Reply:
Just incorporated this change into this PR. @MCWertGaming Feel free to use the same in your PR.

![image](https://user-images.githubusercontent.com/33723045/110226158-e40ce800-7eba-11eb-8bd4-2e5cd1dae831.png)

Reply:
I will do a rebase! I create a pull request later.